docker-compose/Makefile

```diff
@@ -21,12 +24,13 @@ all-clean:
  
 .PHONY: matrixone dev clone and build
 dev-pre:
-	git clone git@github.com:matrixorigin/matrixone.git -b ${BRANCH} && cp Dockerfile matrixone
+	git clone ${REPO} -b ${BRANCH} && cp Dockerfile matrixone
```

Comment:
There is a Dockerfile that existed in https://github.com/matrixorigin/matrixone/blob/main/optools/image_builder/Dockerfile, maybe it's better to use it directly.